### DISCUSSION TITLE
Samotech SM323 retrofit dimmer

### DISCUSSION THREAD

**Discussion by pedrolamas** [Koenkk/zigbee2mqtt#7432] [Devices]
I've been thinking in getting a couple of these Samotech SM323 retrofit dimmers for my place, and I was wondering if anyone has any experience with them?

@avee87 given you added the support for these, what do you think of them? My understanding is that they work with one-wire connection, are you using like that?

**Comment by Mark-Sellwood** [comment]
I'm following this too as I'm looking to replace some existing dimmers with these.

**Comment by avee87** [comment]
They are really good! I installed 3 of them in place of [these](https://www.zigbee2mqtt.io/devices/AU-A1ZB2WDM.html) and couldn't be happier. Previously one of the lights would be partially lit even when the dimmer is off, now there is no such problem. Dimming works well and, unlike my previous dimmers, you don't need to rotate them very far to get full range of dimming.

Yes, these are single wire and that's how I use them.

The only problem is that they don't seem to fit into 2-gang single width faceplate. I'm currently looking for a solution for my other room where this is a constraint...

**Comment by Mark-Sellwood** [comment]
Thanks, on the Samotech website they do show  a 2-gang single width faceplate, one is mounted upside down, don't know why, maybe that's the trick?
I would need the same setup in my lounge, 1 single face plate with 2 dimmers.
https://www.samotech.co.uk/products/zigbee-dimmer-switch-hue-compatible/

**Reply by avee87** [reply]
Well, I tried that configuration and it doesn't work at least for my plate. The thing is that the rotary thing on these dimmers is a bit off-center so in that configuration the distance between them becomes non-standard and they don't align with holes in the plate. I guess I could drill it but would really prefer something cleaner...

**Reply by Mark-Sellwood** [reply]
OK, think I will get 2 & see how I get on.

**Reply by avee87** [reply]
As an update, I bought and cannibalised [this dimmer](https://www.amazon.co.uk/gp/product/B01FK7WTKG/) and managed to fit Samotech on it's faceplate just fine.

**Comment by bartplessers** [comment]
Following. 
Does anybody know about a standard EU formfactor rotary dimmer? I.e. https://ecataleg.be/nl/product/665117

**Comment by Mark-Sellwood** [comment]
@avee87 I know what I've been meaning to ask, why did you change from Aurora  to Samotech?

**Reply by avee87** [reply]
I was unhappy with Aurora because it left some lights partially lit in turned off state. It also doesn't remember state when losing power - i.e., lights always turn on when power is restored. I tried Samotech and it didn't have these problems so I made the switch.
